### ISSUE TITLE
Feature: Dockerfile for central build

### ISSUE THREAD

**Issue by RobertGoellinger** [sylvainhalle/textidote#230]
Thank you so much for this awesome tool!

I have just coded up a little Dockerfile which can be used to run textidote inside the container and host the html in a nginx instance.

This may be an interesting feature for some of you here. 

Kind regards, 
Robert

**Comment by RobertGoellinger** [comment]
Yes absolutely! I tried pushing my proposed changes to a branch on here, but GitHub doesn't want to let me push. 
Is there anything I have to keep in mind?

**Comment by sylvainhalle** [comment]
I suggest you open a pull request with your modifications, which I can then merge with the current branch.

**Comment by aadityabhatia** [comment]
This is great. Would you consider setting up a [GitHub action to automatically publish the docker image](https://docs.github.com/en/actions/publishing-packages/publishing-docker-images#publishing-images-to-github-packages)? I can create a pull request for it, though I would not be able to create the GitHub token needed to make it work.

**Comment by sylvainhalle** [comment]
Yes, could you please draft the pull request?

**Comment by sylvainhalle** [comment]
Any progress on that feature?
